src/cascade/model/compartmental.py

```diff
@@ -93,7 +93,7 @@ def lx(t):
 
 .. math::
 
-    P = \frac{C}{S+C}
+    P = \\frac{C}{S+C}
```

Comment:
When you see this, turn it the comment into raw strings. It's my fault for the error. I think the raw strings make the error very much go away with one character.